### PR TITLE
[Fix] Update GraphQL deprecated reasons

### PR DESCRIPTION
**Pull request by mnigh**
## 👋 Introduction

This PR fixes some of the content of the deprecated reasons and adds issue numbers for work on removing the deprecated queries and fields.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Read through and make sure reasons make sense (unlike me in #7641 🥴)

### CODE REVIEW

File: api/graphql/schema.graphql [hunk 1]
```diff
@@ -893,12 +895,16 @@ type Query {
     @find(model: "User")
     @guard
     @can(ability: "view", query: true, model: "User")
-    @deprecated(reason: "applicant is deprecated. User instead.")
+    @deprecated(
+      reason: "applicant is deprecated. Use User instead. Remove in #7651."
```
Comment (esizer) [inline api/graphql/schema.graphql:899]: Another mistake by me, should be `user` lowercase 😬

**Review by esizer — CHANGES_REQUESTED** [review]
Found one more 😅

File: api/graphql/schema.graphql (hunk 1 above)
Comment (mnigh) [reply api/graphql/schema.graphql]: See 30e27dcc7486fcd735b993f0ffcb4add987b5f35.

**Review by esizer — APPROVED** [review]
Thanks 🙇‍♀️

**Pull request merged**